jaeger-spark-dependencies-elasticsearch/src/main/java/io/jaegertracing/spark/dependencies/elastic/ElasticsearchDependenciesJob.java

```diff
@@ -209,7 +215,27 @@ private static String prefix(String prefix) {
   }
 
   public void run(String peerServiceTag) {
-    run(indexDate("jaeger-span"), indexDate("jaeger-dependencies") ,peerServiceTag);
+
+    String[] readIndices;
+    String[] writeIndex;
```

Comment:
nit: `writeIndices` since it's an array?

Reply:
True but also not really intuitive then.

The data type is an array, just because the `indexDate ` helper does return an array (https://github.com/jaegertracing/spark-dependencies/blob/a41f1734c1ad647f3b501194fd49b12161780c41/jaeger-spark-dependencies-elasticsearch/src/main/java/io/jaegertracing/spark/dependencies/elastic/ElasticsearchDependenciesJob.java#L215). But this is only for read indices.

I gladly clean this up by either splitting the helpers for read and write or by removing (breaking) compatibility with the old `prefixBefore19` format, but it's changing more than the feature I wanted to add. Just let me know that you'd rather like